pkgs/development/python-modules/gurobipy/linux.nix

```diff
@@ -1,21 +1,15 @@
-{ fetchurl, python }:
-assert python.pkgs.isPy27;
+{ buildPythonPackage, python, gurobi }:
 
-python.pkgs.buildPythonPackage
-  { pname = "gurobipy";
-    version = "7.5.2";
-    src = fetchurl
-      { url = "http://packages.gurobi.com/7.5/gurobi7.5.2_linux64.tar.gz";
-        sha256 = "13i1dl22lnmg7z9mb48zl3hy1qnpwdpr0zl2aizda0qnb7my5rnj";
-      };
-    setSourceRoot = "sourceRoot=$(echo gurobi*/*64)";
-    patches = [ ./no-clever-setup.patch ];
-    postInstall = "mv lib/libaes*.so* lib/libgurobi*.so* $out/lib";
-    postFixup =
-      ''
-        patchelf --set-rpath $out/lib \
-          $out/lib/python2.7/site-packages/gurobipy/gurobipy.so
-        patchelf --add-needed libaes75.so \
-          $out/lib/python2.7/site-packages/gurobipy/gurobipy.so
-      '';
-  }
+buildPythonPackage {
+  pname = "gurobipy";
+  version = "9.1.2";
+  src = gurobi.src;
+  setSourceRoot = "sourceRoot=$(echo gurobi*/*64)";
+  patches = [ ./no-clever-setup.patch ];
+  postInstall = "mv lib/libgurobi*.so* $out/lib";
+  postFixup =
+    ''
+      patchelf --set-rpath $out/lib \
+        $out/lib/${python.libPrefix}/site-packages/gurobipy/gurobipy.so
+    '';
+}
```

Comment:
```suggestion
buildPythonPackage {
  pname = "gurobipy";
  version = "9.1.2";

  src = gurobi.src;

  setSourceRoot = "sourceRoot=$(echo gurobi*/*64)";

  patches = [ ./no-clever-setup.patch ];

  postInstall = ''
    mv lib/libgurobi*.so* $out/lib
  '';

  postFixup = ''
    patchelf --set-rpath $out/lib \
      $out/lib/${python.libPrefix}/site-packages/gurobipy/gurobipy.so
  '';
  
  meta = ...
}
```

Reply:
Not really sure of how exactly the metadata ought to look, but maybe that is out of scope of this PR.